### ISSUE TITLE
[BUG] Capitalize the `h` in the `GitHub` word

### ISSUE THREAD

**Issue by Panquesito7** [web3community/web3community.github.io#71]
### Description of the bug

The `h` should be capitalized as the word itself should be `GitHub`. Go to the `src/_includes/partials/footer.html` file. In line 96, change the word to `GitHub`, as currently the `h` is not capitalized.

### Please add screenshots (if applicable)

![image](https://user-images.githubusercontent.com/51391473/135951167-8d65ed5c-5aea-4789-94ee-56f1b87e4e0e.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by durgeshahire07** [comment]
can i work on this?

**Comment by Ashwani132003** [comment]
Done

**Comment by Ashwani132003** [comment]
![Screenshot (232)](https://user-images.githubusercontent.com/76273539/135951543-4a838db3-fb27-49be-a45a-75229dcdc173.png)
check it

**Comment by Panquesito7** [comment]
> ![Screenshot (232)](https://user-images.githubusercontent.com/76273539/135951543-4a838db3-fb27-49be-a45a-75229dcdc173.png) check it

You must fork this repository, implement your changes, make a PR, and then we will review it to see if it's good. 🙂

**Comment by Panquesito7** [comment]
> can i work on this?

Feel free to work on this if you want. I'm just not sure if @Ashwani132003 wants to work on that, though.

**Comment by krishguptadev** [comment]
I can do it?

**Comment by krishguptadev** [comment]
@Panquesito7 I do not think we should add the hacktoberfest label. It's just changing a letter

**Comment by Panquesito7** [comment]
> I can do it?

A PR was already made, sorry. 😅
